torch/_dynamo/compiled_autograd.py

```diff
@@ -91,6 +91,32 @@ def begin_capture(self, inputs: List[torch.Tensor], sizes: List[int]):
         self.stack.enter_context(disable_proxy_modes_tracing(enable_current=True))
         return inputs, sizes
 
+    def proxy_call_backward(
+        self,
+        inputs,
+        output_sizes,
+        saved_tensors,
+        backward_idx: int,
+    ):
+        assert self.hooks_proxy is not None
+        backward_fn = self.hooks_proxy[backward_idx]
+        proxies = self.fx_tracer.create_proxy(
+            kind="call_function",
+            target=call_backward,
+            args=(
+                backward_fn,
+                self.to_proxy(saved_tensors),
+                *self.to_proxy(inputs),
+            ),
+            kwargs={},
+        )
+
+        with disable_proxy_modes_tracing():
+            # create fake Tensors
+            grad_ins = [torch.Tensor(output_size) for output_size in output_sizes]
```

Comment:
This doesn't look right:
```
>>> torch.Tensor([1,2,3])
tensor([1., 2., 3.])
>>> torch.zeros([1,2,3])
tensor([[[0., 0., 0.],
         [0., 0., 0.]]])
>>> 
```

Also, there are no dtype/device/strides.


Please add a test that would have caught this issue.

Reply:
output_size is a torch.Size, the ctor has special handles it properly with default elements. but it is missing the metadata. 

For dtype, it looks like the graph auto casts it with aten._to_copy. I can add a test to check for graph nodes.
```
getitem_3: "f32[3, 3]" = call_backward[0];  call_backward = None
_to_copy: "f16[3, 3]" = torch.ops.aten._to_copy.default(getitem_3, dtype = torch.float16);  getitem_3 = None
view: "f16[9]" = torch.ops.aten.view.default(_to_copy, [9]);  _to_copy = None
```
For device and strides of the fake tensor, it doesn't seem like they affect the runtime result. I'm not sure how I can test for those